src/Microsoft.Extensions.HealthChecks/CachedHealthCheckExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.HealthChecks
+{
+    public static class CachedHealthCheckExtensions
+    {
+        public static ValueTask<IHealthCheckResult> RunAsync(this CachedHealthCheck check, IServiceProvider serviceProvider)
```

Comment:
BTW throughout ASP.NET/EF Core we often have CancellationToken be an optional param w/ default value of `default(CancellationToken)` (which is equivalent to `None`). That could mean this extension method would go away.

Thoughts?